doc/nrf/documentation/build.rst

```diff
@@ -27,6 +27,37 @@ Complete the following steps to install the required tools:
 
 1. If you have not done so already, install the |NCS| as described in :ref:`auto_installation` or :ref:`manual_installation`.
 #. Install or update all required :ref:`Python dependencies <additional_deps>`.
+#. Install the additional Python dependencies for building the documentation:
+
+   .. tabs::
+
+     .. group-tab:: Windows
+
+        Enter the following command in a terminal window in the :file:`ncs/nrf` folder:
+
+        .. parsed-literal::
+           :class: highlight
+
+           pip3 install -U -r doc/requirements.txt
+
+     .. group-tab:: Linux
+
+        Enter the following command in a terminal window in the :file:`ncs/nrf` folder:
+
+        .. parsed-literal::
+           :class: highlight
+
+           pip3 install -U --user -r doc/requirements.txt
+
+     .. group-tab:: macOS
+
+        Enter the following command in a terminal window in the ::file:`ncs/nrf` folder:
+
+        .. parsed-literal::
+           :class: highlight
+
+           pip3 install -U -r doc/requirements.txt
```

Comment:
on venvs, `--user` doesn't need to be used, and `pip3` -> `pip`. So all tabs will contain the same...